frontend/app/lib/interface/input/keyboard.js

```diff
@@ -96,6 +96,7 @@ class Keyboard extends Phaser.Group {
 
   submit() {
     this.submitSignal.dispatch(this.textField.valueSprite.text);
+    super.destroy();
```

Comment:
This still raises a bug, but that's not something to solve here. I just wrote a new ticket #91 to address this issue.